app/Resources/translations/default/messages.xlf

```diff
@@ -2960,4 +2960,86 @@ Comment: $this->list_modules_categories['search_filter']['name'] = $this->l('Sea
       </trans-unit>
     </body>
   </file>
+  <file original="modules/gamification/controllers/admin/AdminGamificationController.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="3d4aafb2eedeba2fbf92e852f0af745a">
+        <source>Merchant Expertise</source>
+        <target>Merchant Expertise</target>
+        <note>Line: 44</note>
+      </trans-unit>
+      <trans-unit id="ca96b4f8d13722aac99da25f94ea1711">
+        <source>Your Merchant Expertise</source>
+        <target>Your Merchant Expertise</target>
+        <note>Line: 18</note>
+      </trans-unit>
+      <trans-unit id="98f770b0af18ca763421bac22b4b6805">
+        <source>Features</source>
+        <target>Features</target>
+        <note>Line: 85</note>
+      </trans-unit>
+      <trans-unit id="f5c7922da355fd289ec1d6469e0583a7">
+        <source>Achievements</source>
+        <target>Achievements</target>
+        <note>Line: 86</note>
+      </trans-unit>
+      <trans-unit id="8189ecf686157db0c0274c1f49373318">
+        <source>International</source>
+        <target>International</target>
+        <note>Line: 87</note>
+      </trans-unit>
+      <trans-unit id="851f12a0c936baace7f0e734d5c624e7">
+        <source>1. Beginner</source>
+        <target>1. Beginner</target>
+        <note>Line: 91</note>
+      </trans-unit>
+      <trans-unit id="583981a16ea761fe852b64094d8a887e">
+        <source>2. Pro</source>
+        <target>2. Pro</target>
+        <note>Line: 92</note>
+      </trans-unit>
+      <trans-unit id="38f7af7416ffcd1524d8a4acda756cbf">
+        <source>3. Expert</source>
+        <target>3. Expert</target>
+        <note>Line: 93</note>
+      </trans-unit>
+      <trans-unit id="e7613fe56cdbeddfc9bb6276fd0f0d12">
+        <source>4. Wizard</source>
+        <target>4. Wizard</target>
+        <note>Line: 94</note>
+      </trans-unit>
+      <trans-unit id="8d03eaad7ff7babdd33c2c74fe479ed0">
+        <source>5. Guru</source>
+        <target>5. Guru</target>
+        <note>Line: 95</note>
+      </trans-unit>
+      <trans-unit id="e4be4f3e3ae4ee9dda6b60815bf774c1">
+        <source>6. Legend</source>
+        <target>6. Legend</target>
+        <note>Line: 96</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="modules/gamification/gamification.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="bacc1bf300527bad9c6ac2d3b875a8d8">
+        <source>Become an e-commerce expert within the blink of an eye!</source>
+        <target>Become an e-commerce expert within the blink of an eye!</target>
+        <note>Line: 58</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="modules/ps_emailsubscription/ps_emailsubscription.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="2df860f82be52de57fc5e3f7ecac9431">
```

Comment:
Same than Theme Custo, the two following strings have been removed from the Crowdin catalog since the 1.7.4 version, perhaps because they already have their own translation. I can see three solutions:

- we can properly localize them and we put them in the _Modules.Emailsubscription.Admin_ file, where they belong
- we keep them as-is because we cannot change domain at this point
- we delete them from the code here, but it could be risky

Reply:
The new domain is managed in v2.4.0, but the one included in 176 is the 2.3.0
So we would need to update the included module, but I'm not sure we should do it in this version. I'm afraid we need to wait for the 177 version

Reply:
Oh and this case is different from ps_themecusto which includes its own translations, this module has no translations embedded so we need to manage them in the core.
By the way even for ps_themecusto although it includes some translations, there are only three languages managed, but I guess it can be managed via the backoffice then?